test/e2e/webdriver/chrome.js

```diff
@@ -13,6 +13,11 @@ class ChromeDriver {
       args.push('--auto-open-devtools-for-tabs')
     }
     const options = new chrome.Options()
+      .addArguments(
+        '--disable-dev-shm-usage',
+        '--headless',
```

Comment:
I don't think we want to run it headless, if possible. I'm concerned it'd behave differently if it doesn't have to render anything. Plus this'd mean our screenshots on failure won't work anymore.

Reply:
Indeed, this is the cause of the `shadowRoot` issue I was just debugging